src/Message/CalculateSalesTaxRequest.php

```diff
@@ -58,6 +58,13 @@
  */
 class CalculateSalesTaxRequest extends AbstractRequest
 {
+    /**
+     * With the Avalara tax engine, a customer ID is required for tax to be calculated.
+     * Vindicia's advice is to pass a dummy ID if you just want to calculate tax
+     * without a specific user.
+     */
+    const DUMMY_CUSTOMER_ID = 'Dummy ID for tax calculation';
```

Comment:
@nickyr let us know what you think of changing this?

Reply:
@parth21shah changing it to what?

Reply:
Something like this const DUMMY_CUSTOMER_FOR_TAX_CALCULATION = dummy123
as mentioned in one of the outdated diff

Reply:
my reply to that was

> I think the name should be something ridiculously long so it is unlikely that it ever clashes with a real user for anything. 

How about I take your name for the const but my value? I'll also replace the spaces with underscores for good measure

Reply:
Sounds good